Full-Stack-Collective/connectient
# Email confirmation

## Description

This PR sends an email once an appointment request has been made.

## Related Issue

closes #29 

## Acceptance Criteria

A confirmation email is sent to the user once an appointment is requested. This email contains all information that was submitted on the form.
A confirmation email is sent to the use once an appointment has been confirmed. This email contains the user's name, appointment type, and the scheduled date and time.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria

This feature, as of now, is built to use the tester's email to send the email to themself. In you env.local file you need an EMAIL and EMAIL_PASS. The email should be your own and the email pass should be an App Password for your gmail account. Here is a link in how to get one: https://support.google.com/mail/answer/185833?hl=en

Once that is set up, submit an appointment request form and you should receive an email from yourself with all of the appointment details that were entered on the form.

To check the email once an appointment is confirmed, login as an admin, select an appointment that has not yet been confirmed and check the confirmed box. You should receive an email that contains the first and last name, the appointment type, and the scheduled date and time for that user.
